cli/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ _Released 2/13/2024 (PENDING)_
 
 - Improved accessibility of the Cypress App in some areas. Addressed in [#28774](https://github.com/cypress-io/cypress/pull/28774).
 
+**Bugfixes:**
+
+- LaunchPad: Stop reopening browsers after closing when clicking on close button. Fixes [#28852](https://github.com/cypress-io/cypress/issues/28852)
```

Comment:
This needs to be moved above the misc section 
```suggestion
- Fixed an issue which caused the browser to relaunch after closing the browser from the Launchpad. Fixes [#28852](https://github.com/cypress-io/cypress/issues/28852).
```

Reply:
Isn't this a Bug fix? I do not understand.

Reply:
The entire section needs to be moved above the misc section.